src/themes/greyVest/ToggleFiltersButton.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import IconButton from './IconButton'
+
+let ToggleFiltersButton = ({ onClick }) => (
```

Comment:
It's used by things using search layout/search editor because once you hide the filters you need one of these buttons somewhere to go back to a basic mode. In the updated grey vest story, it's used conditionally in the results header:

https://github.com/smartprocure/contexture-react/blob/1be8932ea3884e6af1a145912f406e2fbb58df87/stories/imdb/stories/greyVest.js#L309

Reply:
Ah yes you're right I'd forgotten about that.